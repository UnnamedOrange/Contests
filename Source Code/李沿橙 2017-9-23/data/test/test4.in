5
50
475 506 218 586 143 237 489 564 337 410 616 940 272 31 996 591 488 204 10 487 165 869 818 572 186 694 383 749 304 945 738 952 333 563 411 849 516 532 68 607 138 325 281 730 580 460 927 417 722 747 
50
1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49 50 
6
1 200 500 500 500 600 
6
1 2 2 2 3 5 
6
1 2 3 4 4 7 
