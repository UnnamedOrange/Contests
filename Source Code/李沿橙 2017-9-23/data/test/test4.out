10 31 138 143 186 204 218 237 281 304 325 333 383 410 411 460 475 487 488 489 506 532 563 564 572 580 586 591 607 694 730 738 747 749 849 927 940 945 996 952 869 818 722 616 516 417 337 272 165 68 
1 2 4 6 8 10 12 14 16 18 20 22 24 26 28 30 32 34 36 38 40 42 44 46 48 50 49 47 45 43 41 39 37 35 33 31 29 27 25 23 21 19 17 15 13 11 9 7 5 3 
1 200 500 500 600 500 
1 2 2 2 5 3 
1 2 3 4 7 4 
